renovate.json

```diff
@@ -826,7 +815,10 @@
         "Team:Monitoring"
       ],
       "minimumReleaseAge": "7 days",
-      "enabled": true
+      "enabled": true,
+      "matchDepNames": [
+        "@opentelemetry/{/,}**"
```

Comment:
I was confused, in case anyone else is curious, https://docs.renovatebot.com/string-pattern-matching/#example-glob-patterns.  

`@opentelemetry**/*` is probably closer to our expecations but I don't see an issue with the current version